src/org/exoplatform/shareextension/AccountsFragment.java

```diff
@@ -104,7 +104,7 @@ public boolean setViewValue(View view, Object data, String textRepresentation) {
   public void onListItemClick(ListView l, View v, int position, long id) {
     if (accounts != null && position >= 0 && position < accounts.size()) {
       ExoAccount acc = accounts.get(position);
-      if (acc.password != null && !"".equals(acc.password)) {
+      if (acc.isRememberEnabled && acc.isAutoLoginEnabled && acc.password != null && !"".equals(acc.password)) {
```

Comment:
If these checks don't exist, user will always auto login even the remember me and and autoLogin set to off.
About the values is sometimes false, I think because current app keep only a single instance of AccountsFragment, so accounts list will never be reload event when these setting were changed. It's a common issue when current app keep too much static instance of Fragments and Activities.